arns/arns-core-1.md

```diff
@@ -34,12 +35,15 @@ The **ARNS-CORE-1** Specification includes the following requirements for valid,
 
 **Records Table**:
 
-- Must have a `Records` table containing `subDomains` and their associated `transactionId` and `ttlSeconds`.
+- Must have a `Records` table containing `subDomains` and their associated `transactionId`, `ttlSeconds`, and optional `priority`.
 - The `Records` object should be sorted alphabetically.
 - Each Record subdomain must be a string.
 - Each Record must include a `transactionId`, which is where this subdomain will point to.
 - Each Record must include a `ttlSeconds` value, which is the suggested time for caching by ArNS Name Resolvers.
 - The `ttlSeconds` value must not be less than 900 seconds.
+- Each Record may include a `priority` value, which determines the sort order of undernames served by gateways.
+- The `priority` value for the default record `@` must be 0.
+- The `priority` value for other records must be an integer greater than 0 or nil.
```

Comment:
Was confused by this sentence for a second because I thought it was implying nil was an integer.. punctation there helps i think
```suggestion
- The `priority` value for other records must be an integer greater than 0, or nil.
```